StripeCore/StripeCore/Source/Helpers/FeatureFlags.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FeatureFlags.swift
+//  StripeCore
+//
+//  Created by Till Hellmund on 4/26/25.
+//
+
+import Foundation
+
+@_spi(STP) public enum FeatureFlags {
+    // Add new features here to enable them in debug builds and disable them in release builds
```

Comment:
Kind of nervous about this, because our tests always run as debug builds. If we want a central location for these, I'd rather have them be explicitly configurable

Reply:
Replaced it with an `@_spi(STP)` annotated field on `PaymentSheet`.